raccoongang/edx-platform
# translation fix

**Description:** Translation fix for report 'Download course users'

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-102

**Testing instructions:**

1. Open page /sysadmin/courses
2. Fill the form above the button 'Download course users ' with correct course id
3. Press 'Download course users'
4. If first column header is not 'username'  - check failed. 
5. Switch to ru localization of cite 
6. If name of the button is not 'Скачать список пользователей курса' - check failed.

**Reviewers:**
- [@RuslanSkiraRaccoongang  ] tag reviewer


**Merge checklist:**


**Author concerns:** exception blocks state is temporary and will be updated later
